src/Error/Debugger.php

```diff
@@ -32,6 +33,16 @@
  */
 class Debugger
 {
+    use InstanceConfigTrait;
```

Comment:
I still can't help but think that we should be trying to remove the singleton features instead of add more. Given that we're just adding one config option would static methods to set/get the masked values be reasonable?